### ISSUE TITLE
Capitalization Typos in line 77 - Logistic Regression

### ISSUE THREAD

**Issue by jacksusank** [rtealwitter/datamining2026spring#33]
Capitalization Typos:

Old Line: "We have shown that Minimizing Binary Cross-Entropy is mathematically identical to finding the Maximum Likelihood Estimate for our data under a Bernoulli distribution. This provides a rigorous statistical footing for our choice of loss function."

Proposed Fix: "We have shown that minimizing binary cross-entropy is mathematically identical to finding the Maximum Likelihood Estimate for our data under a Bernoulli distribution. This provides a rigorous statistical footing for our choice of loss function."

https://github.com/rtealwitter/datamining2026spring/blob/4ab71393f59e05fd1ad92b1a427756007888f03a/notes/LogisticRegression.qmd#L77

**Comment by rtealwitter** [comment]
Lol this is stylistic
